tests/test_gbdesAstrometricFit.py

```diff
@@ -1058,6 +1061,114 @@ def test_inputCameraModel(self):
                     self.assertAlmostEqual(np.std(dDec), 0)
 
 
+class TestApparentMotion(lsst.utils.tests.TestCase):
+    """This class simulates an array of objects with random proper motions and
+    parallaxes and compares the results of
+    `lsst.drp.tasks.gbdesAstrometricFit.calculate_apparent_motion` against the
+    results calculated by astropy."""
+
+    @classmethod
+    def setUpClass(cls):
+        np.random.seed(12345)
+
+        cls.RAs = np.random.rand(10) + 150.0
+        cls.Decs = np.random.rand(10) + 2.5
+
+        cls.pmRAs = np.random.random(10) * 10
+        cls.pmDecs = np.random.random(10) * 10
+
+        cls.parallaxes = abs(np.random.random(10) * 5)
+
+        cls.mjds = np.random.rand(10) * 20 + 57000
+
+    def testProperMotion(self):
+        """Calculate the change in position due to proper motion for a given
+        time change, and compare the results against astropy.
+        """
+
+        parallaxes = np.zeros(len(self.RAs))
```

Comment:
I'd add a comment here that this test is not of the parallax correction, so it's more obvious why this is zero.
```suggestion
        parallaxes = np.zeros_like(self.RAs)
```

Reply:
`np.zeros_like` is much slower that `np.zeros`. Not a big deal here because `self.RAs` is not big, but I usually avoid zeros_like and ones_like.

Reply:
Oh, huh, I didn't know that there was a speed difference.